module_utils/ansible_nailgun_cement.py

```diff
@@ -10,6 +10,7 @@
     AbstractComputeResource,
     AbstractContentViewFilter,
     ActivationKey,
+    AuthSourceLDAP,
```

Comment:
Sorry, i completely overlooked, that you are not going to use this file at all in the module.

Reply:
Restored the file.